### PR TITLE
Switched HP to its own import system

### PR DESCRIPTION
**Pull request by matentzn**
This pull request does three things:
1) It adds a new directory **/imports** into the src/ontology directory which contains all HP imports, freshly created based on the latest version of HP
2) It changes the import declarations inside of hp-edit.owl from the previous upheno ones to the new hp-tailored imports. the catalog-0.0.1.xml will automatically redirect the imports locally if you open hp-edit with Protege or ROBOT
3) It adds a few entries to .gitignore

@drseb Please review as see fit. I think its best to look at the github diff (esp. hp-edit.owl), and as a second check, check out the hp-imports branch locally and try to open hp-edit.owl. You should see the cleaned up hierarchy now. As a second check, click, in Protege, on active ontology and check whether your chebi import (for example), looks something like this: 
image

Lets be careful with this one: I have triple checked everything, but one or two more sets of eyes would be great I think!

### CODE REVIEW

**Comment by matentzn** [comment]
Okay changing the default-catalog.xml fixed the above. However, now we are not passing by your OBO related QC. I need help from you to fix this; Do I interpret the travis log correctly in that the failure was caused by the fact that obsolete classes were used in some axioms? This would be extremely hard to fix; Since all of the occurrences appear to be outside of HP. Can reduce the status of these occurences to WARNING?

**Comment by drseb** [comment]
I think it fails because there is no `hp.obo` file in `src/ontology/`

**Comment by matentzn** [comment]
Yes, there is no hp.obo I think because the qc before that fails... Because of these errors (examples):

```
Obsolete class GO:0044708 in axiom: SubClassOf(GO:0007626 'locomotory behavior' GO:0044708 GO:0044708)
		Obsolete class GO:0044708 in axiom: SubClassOf(GO:0007622 'rhythmic behavior' GO:0044708 GO:0044708)
		Obsolete class GO:0044699 in axiom: SubClassOf(GO:0048511 'rhythmic process' GO:0044699 GO:0044699)
```

**Comment by drseb** [comment]
yes. hp.obo is generated from hp-edit. and then I run QC on the generated hp.obo. So we need to fix NBO before we can continue?

File: src/ontology/Makefile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-all: hp.obo hp.owl all-subsets hp-inferred.obo
+all: hp.obo hp.owl hp-inferred.obo
```
Comment (drseb) [inline src/ontology/Makefile]: @cmungall please comment if this change has consequences that we are maybe not aware of. thanks

File: src/ontology/Makefile [hunk 2]
```diff
@@ -17,11 +17,11 @@ catalog-v001.xml: default-catalog.xml
 
 # Note: we currently use no-reasoner for now, until we can trust all inferences
 build/hp-simple.obo: hp-edit.owl catalog-v001.xml
-	ontology-release-runner --catalog-xml catalog-v001.xml $(OORT_ARGS) --ignoreLock --skip-release-folder --outdir build --simple --allow-overwrite --no-reasoner $<
+	ontology-release-runner --catalog-xml catalog-v001.xml $(OORT_ARGS) --ignoreLock --no-subsets --skip-release-folder --outdir build --simple --skip-ontology-checks --allow-overwrite --no-reasoner $<
```
Comment (drseb) [inline src/ontology/Makefile]: @cmungall please comment if this change has consequences that we are maybe not aware of. thanks

Comment (cmungall) [reply src/ontology/Makefile]: Should be fine. I think you should get rid of oort asap

Comment (matentzn) [reply src/ontology/Makefile]: I have already replaced all this in another pull request, but we need more testing before we can merge that one. @drseb seems its fine.

**Pull request merged**